site/docs/v1/tech/admin-guide/licensing.adoc

```diff
@@ -103,3 +104,11 @@ You should use the non-production license Id in your Kickstart files.
 include::docs/v1/tech/installation-guide/_kickstart-license-text.adoc[]
 
 Learn more about link:/docs/v1/tech/installation-guide/kickstart[setting up Kickstart].
+
+== Advanced Scenarios
+
+If you are running FusionAuth with limited or no network access, you can do so using an air gapped configuration. The license text will be available in your https://account.fusionauth.io/account[Account]. Include this text in addition to the license Id when activating. If you need an air gapped license, please link:/contact[contact sales] for more information. 
```

Comment:
Note that this will require this feature to be enabled by FusionAuth and generally speaking is going to require an Enterprise license, and is still only enabled on a case by case basis.

Reply:
Agreed. I think it is a conversation sales will be glad to have.